CMakeLists.txt

```
@@ -696,14 +696,33 @@ hide_variable(GMLC_UTILITIES_STATIC_LIB)
 hide_variable(GMLC_UTILITIES_CLANG_TIDY_OPTIONS)
 
 target_compile_definitions(gmlc_utilities PUBLIC USE_STD_STRING_VIEW=1)
-add_library(HELICS::utilities ALIAS gmlc_utilities)
+
 set_target_properties(gmlc_utilities PROPERTIES FOLDER Extern)
 if(NOT HELICS_DISABLE_BOOST)
     target_include_directories(
         gmlc_utilities SYSTEM PRIVATE $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
     )
 endif()
 
+# -----------------------------------------------------------------------------
+# create networking target
+# -----------------------------------------------------------------------------
+
+set(GMLC_NETWORKING_INSTALL OFF CACHE INTERNAL "")
+set(GMLC_NETWORKING_ASIO_INCLUDE ${PROJECT_SOURCE_DIR}/ThirdParty/asio/asio/include)
+set(GMLC_NETWORKING_CONCURRENCY_INCLUDE ${PROJECT_SOURCE_DIR}/ThirdParty/concurrency)
+set(GMLC_NETWORKING_DISABLE_ASIO ${HELICS_DISABLE_ASIO})
+add_subdirectory(ThirdParty/networking)
+
+hide_variable(GMLC_NETWORKING_GENERATE_DOXYGEN_DOC)
+hide_variable(GMLC_NETWORKING_INCLUDE_BOOST)
+hide_variable(GMLC_NETWORKING_WITH_CMAKE_PACKAGE)
+hide_variable(GMLC_NETWORKING_OBJECT_LIB)
+hide_variable(GMLC_NETWORKING_STATIC_LIB)
+hide_variable(GMLC_NETWORKING_CLANG_TIDY_OPTIONS)
+
+set_target_properties(gmlc_networking PROPERTIES FOLDER Extern)
```

Can the namespaced alias target name be used here?

```suggestion
set_target_properties(gmlc::networking PROPERTIES FOLDER Extern)
```